### PR TITLE
INTLY-2832 move adding walkthroughs to user doc

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+. When the deployment is complete, refresh the Webapp in your Browser.
++
+You should now see new walkthroughs available from the dashboard.
++
+[NOTE]
+====
+If the git repository is updated with new content, the web app is not automatically updated. You must:
+
+. Open the link:{openshift-host}[Openshift Console].
```
Comment (briandooley) [inline modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc]: Is this link correct?  The link resolves to an "openshift-host" file in the partials directory but can't see any sign of it in this PR.

**Review by briandooley — COMMENTED** [review]
All looks good apart from the possible wrong link.

File: modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc (hunk 1 above)
Comment (pwright) [reply modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc]: oops, updated

**Review by briandooley — APPROVED** [review]
looks grand.

**Pull request merged**